src/app/components/student/student.component.ts

```diff
@@ -12,11 +14,14 @@ export class StudentComponent implements OnInit {
   students: Student[] = [];
   student = Student;
   selectedStudent: Student;
+  modalRef: BsModalRef;
+  idForDeletion: string;
```

Comment:
please add some meaningful name for ex. `deleteStudentId`, because you don't delete ID, you delete student